src/sota_tools/ostree_object_test.cc

```diff
@@ -12,7 +12,9 @@
 
 std::string port;
 
-/* Verify that constructor does not accept a nonexistent repo. */
+/* Verify that constructor does not accept a nonexistent repo.
+ *
+ * Note that this will not fail on release builds! */
```

Comment:
Good catch, but isn't that a bit bad, for example if we want to run our test suite on qemu? This could be a FIXME/TODO.

Reply:
Yeah, I'm actually working on fixing that right now! My current solution is to wrap it around `NDEBUG` so that it doesn't run on Release builds.

Reply:
Ok but what does it test then? It's not the actual behaviour of the class.

Reply:
Currently it fails in debug builds because of an assert that checks that the repo is valid. Would you prefer that instead threw an exception so it failed in all cases?

Reply:
Either that or just remove this test. If we just test that the constructor has an assert, I don't think it brings much value.